src/calculator.m

```diff
@@ -0,0 +1,22 @@
+classdef Calculator
+    methods(Static)
+        function result = add(a, b)
+            result = a + b;
+        end
+        
+        function result = subtract(a, b)
+            result = a - b;
+        end
+        
```

Comment:
Similar to the add method, input validation for the subtract method should be considered.